relayer/src/chain/cosmos/types/events/client.rs

```diff
@@ -1,125 +0,0 @@
-use ibc::core::ics02_client::error::Error;
-use ibc::core::ics02_client::events::{
-    Attributes, ClientMisbehaviour, CreateClient, UpdateClient, UpgradeClient,
-    CLIENT_ID_ATTRIBUTE_KEY, CLIENT_TYPE_ATTRIBUTE_KEY, CONSENSUS_HEIGHT_ATTRIBUTE_KEY,
-    HEADER_ATTRIBUTE_KEY, HEIGHT_ATTRIBUTE_KEY,
-};
-use ibc::core::ics02_client::header::AnyHeader;
-use ibc::events::{IbcEvent, IbcEventType};
-use tendermint::abci::Event as AbciEvent;
-
-pub fn try_from_tx(event: &AbciEvent) -> Option<IbcEvent> {
-    match event.type_str.parse() {
-        Ok(IbcEventType::CreateClient) => extract_attributes_from_tx(event)
-            .map(CreateClient)
-            .map(IbcEvent::CreateClient)
-            .ok(),
-        Ok(IbcEventType::UpdateClient) => match extract_attributes_from_tx(event) {
-            Ok(attributes) => Some(IbcEvent::UpdateClient(UpdateClient {
-                common: attributes,
-                header: extract_header_from_tx(event).ok(),
-            })),
-            Err(_) => None,
-        },
-        Ok(IbcEventType::ClientMisbehaviour) => extract_attributes_from_tx(event)
-            .map(ClientMisbehaviour)
-            .map(IbcEvent::ClientMisbehaviour)
-            .ok(),
-        Ok(IbcEventType::UpgradeClient) => extract_attributes_from_tx(event)
-            .map(UpgradeClient)
-            .map(IbcEvent::UpgradeClient)
-            .ok(),
-        _ => None,
-    }
-}
-
-fn extract_attributes_from_tx(event: &AbciEvent) -> Result<Attributes, Error> {
```

Comment:
IMHO, this code should continue to live in the relayer crate because module users wouldn't be interested in conversions from `AbciEvent` to `IbcEvent`.